project2/shared-libraries/dao/src/main/java/com/revature/dao/BusinessCardDAO.java

```diff
@@ -1,6 +1,6 @@
-package com.revature.businessCard.dao;
+package com.revature.dao;
```

Comment:
Small nitpick, but this folder should be plural, as in `daos` instead of `dao`...

Reply:
Fair enough. It'll have to be changed in multiple places.

Reply:
Intellij's right click -> 'Refactor' will actually handle it being changed in all necessary locations! I thought common convention was to use singular package names in Java though

Reply:
Typically Java packages should use plural names for packages with homogeneous contents and singular names for packages with heterogeneous contents. In our case, the `dao` package will hold several similar `dao.java` files. Thus the recommendation for a plural name.

However, it doesn't necessarily matter for our project. Any name/naming convention will work just fine. That's why it's just a nitpick and the PR was still approved. This change would exist only to reinforce best development practices, and I suppose help satisfy the OCD of the group.

Reply:
Ahhh that makes sense. I had no idea, thanks for explaining!